hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/DatanodeAdminMonitorImpl.java

```diff
@@ -82,6 +85,44 @@ public class DatanodeAdminMonitorImpl implements DatanodeAdminMonitor {
   private long unhealthyContainers = 0;
   private long underReplicatedContainers = 0;
 
+  @SuppressFBWarnings(value = "SIC_INNER_SHOULD_BE_STATIC")
+  private final class ContainerStateInWorkflow {
+    private long sufficientlyReplicated = 0;
+    private long unhealthyContainers = 0;
+    private long underReplicatedContainers = 0;
+    private String host = "";
+
+    private ContainerStateInWorkflow(String host,
+                                     long sufficientlyReplicated,
+                                    long underReplicatedContainers,
+                                    long unhealthyContainers) {
+      this.host = host;
+      this.sufficientlyReplicated = sufficientlyReplicated;
+      this.unhealthyContainers = unhealthyContainers;
+      this.underReplicatedContainers = underReplicatedContainers;
+    }
+
+    public void setAll(long sufficiently,
+                    long under,
```

Comment:
Formatting here seems off again - should either be 4 spaces in from the line above or aligned with the other parameters.

Reply:
Fixed.